src/client/components/organism/navbar/index.tsx

```diff
@@ -2,42 +2,50 @@ import classNames from 'classnames';
 import { Link } from 'react-router-dom';
 import './styles.scss';
 import Logo from '../../atoms/logo';
-import { useContext } from 'react';
+import { useContext, useState } from 'react';
 import StateContext from '../../../utils/stateContext';
+import BurgerButton from '../../atoms/burgerButton';
+import useIsMobile from '../../../utils/useIsMobile';
 
 type NavbarProps = {
   isSticky: boolean;
 };
 
 const Navbar = ({ isSticky }: NavbarProps) => {
   const { user, handlers } = useContext(StateContext);
+  const isMobile = useIsMobile();
+
   return (
-    <div className={classNames('navbar', { 'navbar--sticky': isSticky })}>
-      <Link to="/" className="navbar__logo">
-        <Logo animate={isSticky} dark />
-      </Link>
-      <div className={classNames('navbar__links', { 'navbar__links--sticky': isSticky })}>
-        <Link className="navbar__link" to="/features">
-          FEATURES
-        </Link>
-        <Link className="navbar__link" to={`/${user?.userName}`}>
-          ABOUT
-        </Link>
-        <Link className="navbar__link" to="/edit">
-          Edit Portfolio
+    <>

```

Comment:
I think there is no need for this React Fragment. the block(container) in already wrapping everything